docs/user/resources/04-80-azure-redis-instance.md

```diff
@@ -0,0 +1,72 @@
+
+> [!Note] Non SSL port is disabled.
+
+# Specification
+
+This table lists the parameters of AzureRedisInstance, together with their descriptions:
+
+| Parameter                                               | Type   | Description                                                                                                                                                                                                                             |
+|---------------------------------------------------------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **ipRange**                                             | object | Optional. IpRange reference. If omitted, the default IpRange is used. If the default IpRange does not exist, it will be created.                                                                                                        |
+| **ipRange.name**                                        | string | Required. Name of the existing IpRange to use.                                                                                                                                                                                          | 
+| **sku.capacity**                                        | int    | Required. The service capacity of the instance. Supported values are 1, 2, 3, and 4. Note that tear is 'P' - Premium.                                                                                                                   |
+| **redisVersion**                                        | int    | Optional. The version of Redis software. Defaults to `6.0`.                                                                                                                                                                             |
+| **shardCount**                                          | int    | Optional. Number of shards.                                                                                                                                                                                                             |
+| **replicasPerPrimary**                                  | int    | Optional. Number of replicas, for `Premium` tier it is [up to 3.](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-high-availability)                                                                                |
+| **redisConfiguration**                                  | object | Optional.                                                                                                                                                                                                                               |
```

Comment:
Can you also add a description for **redisConfiguration**? Here, it only says "Optional".